CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Confluent's Python client for Apache Kafka
 
+## v1.7.0
+
+v1.7.0 is a maintenance release with the following fixes and enhancements:
+
+- Add note about Windows environment variables (@slominskir, #1094).
```

Comment:
I missed the review for last PR, will remove this when create the final release tag